arboretum/permissions/fetchers/github/fetch_org_permissions.py

```diff
@@ -0,0 +1,62 @@
+# -*- mode:python; coding:utf-8 -*-
+# Copyright (c) 2021 IBM Corp. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Github organization collaborators and forks fetcher."""
+import json
+
+import arboretum.permissions.fetchers.github.fetch_org_collaborators as collabs
+
+from compliance.evidence import DAY, RawEvidence, raw_evidence
+from compliance.utils.data_parse import get_sha256_hash
+from compliance.utils.services.github import Github
+
+
+class GithubOrgPermissionsFetcher(collabs.GithubOrgCollaboratorsFetcher):
+    """Fetch Github repository colaborators and forks."""
```

Comment:
```suggestion
    """Fetch Github repository permissions evidence."""
```

BTW you should turn auto spell check on in your IDE or editor (collaborators is misspelled).  It would make the review go smoother.